docker/release/config/opensearch/opensearch-docker-entrypoint.sh

```diff
@@ -58,7 +58,7 @@ function setupPerformanceAnalyzerPlugin {
             echo "Disabling execution of $OPENSEARCH_HOME/bin/$PERFORMANCE_ANALYZER_PLUGIN/performance-analyzer-agent-cli for OpenSearch Performance Analyzer Plugin"
         else
             echo "Enabling execution of OPENSEARCH_HOME/bin/$PERFORMANCE_ANALYZER_PLUGIN/performance-analyzer-agent-cli for OpenSearch Performance Analyzer Plugin"
-            $OPENSEARCH_HOME/bin/opensearch-performance-analyzer/performance-analyzer-agent-cli > $OPENSEARCH_HOME/logs/PerformanceAnalyzer.log 2>&1 &
+            $OPENSEARCH_HOME/bin/opensearch-performance-analyzer/performance-analyzer-agent-cli > $OPENSEARCH_HOME/logs/PerformanceAnalyzer.log 2>&1 & disown
```

Comment:
thanks @peterzhuamazon does `disown` remove the job? the idea is to run the `performance-analyzer-agent-cli` outputting the logs to `$OPENSEARCH_HOME/logs/PerformanceAnalyzer.log`?

Reply:
should it be `disown -h`?

Reply:
Nope, `& disown` is very similar to `nohup` in the sense of avoid getting interrupted by `SIGHUP`.
`nohup` will still have the process in job list while `disown` will remove it.

In this case since it is a non-interactive shell anyway we wont really need to bring it from background to foreground, both `nohup` and `& disown` satisfy the use case right here.

I can change it to nohup if you want to, doesnt hurt.

Reply:
I think one advantage of this is, nohup will have a new pid while disown make sure the pid given is the actual process pid.

Reply:
`-h` is not needed as it requires a jobid.
Plus, we are ok if someone specifically send a `SIGHUP` signal to PA, but not when the shell is detached.
The current implementation satisfied our need. Thanks.

Reply:
Thanks, approved peter, `nohup` also adds a `nohup.out` file, just in case if we need that file.

Reply:
> Thanks, approved peter, `nohup` also adds a `nohup.out` file, just in case if we need that file.

Since we are already redirect the log to another file, the `nohup.out` will just not create and redirect there.